FirebaseRemoteConfig/Sources/FIRRemoteConfigComponent.m

```diff
@@ -24,6 +24,28 @@
 
 @implementation FIRRemoteConfigComponent
 
+static NSMutableDictionary<NSString *, FIRRemoteConfigComponent *> *_componentInstances = nil;
+
++ (FIRRemoteConfigComponent *)getComponentForApp:(FIRApp *)app {
+  @synchronized(_componentInstances) {
+    // need to init the dictionary first
+    if (!_componentInstances) {
+      _componentInstances = [[NSMutableDictionary alloc] init];
+    }
+    if (![_componentInstances objectForKey:app.name]) {
+      _componentInstances[app.name] = [[self alloc] initWithApp:app];
+    }
+    return _componentInstances[app.name];
+  }
+  return nil;
+}
+
++ (void)clearAllComponentInstance {
```

Comment:
Is this only for testing? Might add a comment if that's the case

Reply:
Comment added to header file